must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -147,3 +147,25 @@ echo "collecting dump of oc get volumereplication all namespaces" | tee -a  "${B
 echo "collecting dump of oc get volumereplicationgroups all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
 { oc get volumereplicationgroups --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/vrg_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect vrg --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# Collect logs for Ramen DR Hub cluster Resources
+isMirrorPeer=$(timeout 60 oc get crd | grep mirrorpeers |awk '{print $1}')
+if [ "${isMirrorPeer}" == "mirrorpeers.multicluster.odf.openshift.io" ]; then
+            # For Channels of all namespaces
+            { oc get channel --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/channel_all_namespaces"
```

Comment:
Technically collecting Channels, Subscriptions is useful, but I would assume this is an ACM must gather responsibility. We can stop at collecting DRPlacementControl and the PlacementRule.

I am not sure why we collect ManagedClusterAddOns here, am I missing something?

Reply:
Not sure about the ACM must-gather, Collected it here as you mentioned this also needs to be collected. I don't know who is working on ACM must-gather, if there is anyone who can work on this in ACM must-gather, we can close this PR and rather use ACM must-gather. @ShyamsundarR please confirm this.

Reply:
> Not sure about the ACM must-gather, Collected it here as you mentioned this also needs to be collected. I don't know who is working on ACM must-gather, if there is anyone who can work on this in ACM must-gather, we can close this PR and rather use ACM must-gather. @ShyamsundarR please confirm this.

We still need this, one option is to add all of this to the ACM must-gather for the hub. There is no one working on that, if we want to add these to the ACM hub must-gather **we** would need to add the same in there: https://github.com/stolostron/must-gather

Reply:
okay sure. Then I would close this PR and will add it to https://github.com/stolostron/must-gather.

Reply:
I would suggest we check if we can add it in there and then close this PR. Also, need to understand the release cycles for ACM must-gather such that we can be in sync.